config/rbac/role.yaml

```diff
@@ -22,7 +22,6 @@ rules:
   - ""
   resources:
   - secrets
-  - configmaps
```

Comment:
Notification Manager needs the authority of `configmap` to load the template. Maybe we should add configmap [here](https://github.com/kubesphere/notification-manager/blob/master/controllers/notificationmanager_controller.go#L67)?